### ISSUE TITLE
Image uploads are broken

### ISSUE THREAD

**Issue by patbl** [tog22/eaforum#68]
Here's what it looks like:

![brokenuploads](https://cloud.githubusercontent.com/assets/1735266/17086466/9603d4ba-51a7-11e6-8871-6e1754f779f9.png)

When I try to use it in dev with my personal AWS credentials for S3 (`S3KEY_ID` and `S3SECRET_KEY` in `development.ini`), the upload fails with the message 

``` xml
<?xml version="1.0" encoding="UTF-8"?>
<Error><Code>AccessDenied</Code><Message>Access Denied</Message><RequestId>306DF26A77FF1B4F</RequestId><HostId>JLpbix0T1FwwoBnJUqRUmdz7LSXIdN3Y7pcfL8coVMrvQRgMdQ8/SYyTc2iVPc+mN3alYjubHJY=</HostId></Error>
```

I'm not sure whether this error is causing the problem in production as well. The result looks the same.

People are still able to insert externally hosted images (as in these posts: http://effective-altruism.com/ea/zj/the_giving_game_projects_annual_report/, http://effective-altruism.com/ea/zo/scientific_charity_movement/).

**Comment by peterhurford** [comment]
@patbl Is this still an issue?

**Comment by patbl** [comment]
@peterhurford Yes, sort of. When I upgraded TinyMCE, I disabled the image uploading feature. So now it's a missing feature rather than a broken one. It still seems like something worth having—it's annoying to have to upload images on a different site and link to them, and the links are more likely to break that way.
